src/dataMap/metaManager/index.js

```diff
@@ -0,0 +1,248 @@
+import GlobalMeta from './metaLayers/globalMeta';
+import TableMeta from './metaLayers/tableMeta';
+import ColumnMeta from './metaLayers/columnMeta';
+import CellMeta from './metaLayers/cellMeta';
+
+/**
+ * With the Meta Manager class, it can be possible to manage with meta objects for different layers in
+ * one place. All coordinates used to fetch, updating, removing, or creating rows or columns have to
+ * be passed as physical values.
+ *
+ * The diagram of the meta layers:
+ * +-------------+
+ * │ GlobalMeta  │
+ * │ (prototype) │
+ * +-------------+\
+ *       │         \
+ *       │          \
+ *      \│/         _\|
+ * +-------------+    +-------------+
+ * │ TableMeta   │    │ ColumnMeta  │
+ * │ (instance)  │    │ (prototype) │
+ * +-------------+    +-------------+
+ *                         │
+ *                         │
+ *                        \│/
+ *                    +-------------+
+ *                    │  CellMeta   │
+ *                    │ (instance)  │
+ *                    +-------------+
+ *
+ * A more detailed description of the specific layers can be found in the "metaLayers/" modules description.
+ *
+ * @type {MetaManager}
+ */
+export default class MetaManager {
+  constructor(customSettings = {}) {
+    /**
+     * @type {GlobalMeta}
+     */
+    this.globalMeta = new GlobalMeta();
```

Comment:
Maybe this custom settings could be passed to the constructor of the `GlobalMeta` class?

Reply:
I wanted to be consistent with other layers where they accept the higher layer in the constructor. The GlobalMeta is the highest layer so it has no object to inherit from.